gensrc/thrift/AgentService.thrift

```diff
@@ -135,6 +135,7 @@ struct TAlterTabletReqV2 {
     8: optional TTabletType tablet_type
     9: optional i64 txn_id
     10: optional TAlterTabletMaterializedColumnReq materialized_column_req
+    11: optional i64 job_id
```

Comment:
why add job_id?

Reply:
> why add job_id?

Because I want to use job id to trace the complete alter execution path. I can grep `job_id` to trace the all tablet doing alter task easily.